### DISCUSSION TITLE
Go to predefined point using service call.

### DISCUSSION THREAD

**Discussion by agturcz** [PiotrMachowski/lovelace-xiaomi-vacuum-map-card#515] [Q&A]
I've a predefined point in the configuration and I can go to that point using on-screen menu.

How it is possible to go to that point using an automatisation? What service I should use? xiaomi_miio.vacuum_goto asks for X,Y coordinates. Is that possible to use the name/id of the defined point?

**Comment by PiotrMachowski** [comment]
[Here](https://github.com/PiotrMachowski/lovelace-xiaomi-vacuum-map-card/blob/master/docs/templates/setup.md) you have a description how to get details of selected point, to get a service call just click "Copy service call" button instead of "Copy".

It's not possible to use name/id unless you have a vacuum that supports it (e.g. flashed with Valetudo)

**Reply by eskalator** [reply]
Sorry to dig up an old thread, but my question is about something similiar. On Your demo You've got predefined poit "Emptying". I would like to use that functionality from automation, but when i try to send the vacuum to the predefined point it goes there and starts point cleaning. It is not surprising as it is uses vacuum_go_to service(i use miio2). As i understand in miio2(used for STYTJ02YM) vacuum_go_to sends the vacuum to clean around given coordinates. Have You found a way to just go to the specified point and wait? I was wondering if it can be done using status change, but there is no status for "going to point" so i have no idea how to check if the vacuum arrived on the designated coordinates and stop/pause it there. Any thoughts? :)

**Reply by PiotrMachowski** [reply]
@eskalator Potentially you can check coordinates using Map Extractor and stop the vacuum if it is close enough to the desired point, but it highly depends on delays etc

**Reply by eskalator** [reply]
It's a shame that there is no explicit service to only drive the vacuum without cleaning or that there is no separate status for traversing to specified point... 
So either You have different vacuum or this "Emptying" point in Your demo is behaving like in my case :)
Thanks for thoughts anyway.

**Reply by PiotrMachowski** [reply]
Yeah, my vacuum (Roborock S5) just goes to point
